cli/src/commands/test-validator/index.ts

```diff
@@ -10,22 +10,32 @@ class SetupCommand extends Command {
   }
 
   static flags = {
-    without_indexer: Flags.boolean({
+    "without-indexer": Flags.boolean({
       char: "i",
       description: "Runs a test validator without indexer service.",
       default: false,
     }),
-    without_prover: Flags.boolean({
+    "without-prover": Flags.boolean({
       char: "p",
       description: "Runs a test validator without prover service.",
       default: false,
     }),
-    skip_system_accounts: Flags.boolean({
+    "skip-system-accounts": Flags.boolean({
       char: "s",
       description:
         "Runs a test validator without initialized light system accounts.",
       default: false,
     }),
+    "prove-compressed-accounts": Flags.boolean({
+      description: "Enable proving of compressed accounts.",
+      default: true,
+      exclusive: ["without-prover"],
+    }),
+    "prove-new-addresses": Flags.boolean({
+      description: "Enable proving of new addresses.",
+      default: false,
```

Comment:
I think this should be true by default as well, otherwise many people are going to be stuck immediately.